services/s3/types.go

```diff
@@ -316,3 +316,13 @@ type ListObjectsV2Output struct {
 	Prefix                *string
 	StartAfter            *string
 }
+
+type InvalidRangeError struct {
```

Comment:
Does this need to live in awserrors?

Reply:
I think this is fine, this seems like an s3-specific quirk